react-app/src/components/DB.js

```diff
@@ -64,7 +64,9 @@ function DB() {
             Trunk will go here
           </Box> */}
           <Box>
-            <DB_Trunk />
+            {allUserCards &&
```

Comment:
@B-Salinas `allUserCards` refers to all the user's cards, i.e. all cards in user's trunk